migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/ValueTypeLookup.java

```diff
@@ -900,4 +914,16 @@ public void resolveAll(Run run) throws IOException {
     }
   }
 
+  public void resolveAll(Project project) {
+    ReferenceGenome referenceGenome = resolve(project.getReferenceGenome());
+    project.setReferenceGenome(referenceGenome);
+  }
+
+  @VisibleForTesting
+  ReferenceGenome resolve(ReferenceGenome referenceGenome) {
```

Comment:
Add Javadoc as well. Lookup is only possible by alias, whereas most types can be found by ID as well